app/Models/User.php

```diff
@@ -410,6 +412,19 @@ public function manager()
         return $this->belongsTo(self::class, 'manager_id')->withTrashed();
     }
 
+    /**
+     * Establishes the user -> managed users relationship
+     *
+     * @author A. Gianotto <snipe@snipe.net>
+     * @since [v6.4.1]
+     * @return \Illuminate\Database\Eloquent\Relations\Relation
+     */
+    public function managesUsers()
```

Comment:
This should be `managedUsers` I think

```suggestion
    public function managedUsers()
```

Reply:
I don’t think so. I wanted to change the existing managedLocations to managesLocations, but that seemed out of scope.

Reply:
To me, it feels more conversational. “$User manages these users”. It doesn’t really matter, but it was a deliberate choice I made in this case. I figured I’d update the locations method name in a future PR.

Reply:
Huh...that's interesting. This made me think a bit but I realized I think of it in the past tense: "give me the managed locations of this user". 👍🏾